### ISSUE TITLE
Identify source of messages in stderr

### ISSUE THREAD

**Issue by adamralph** [adamralph/simple-exec#140]
A program printing to stderr [should identify itself](https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout) with a prefix. This feature introduces a default prefix of the entry assembly name. E.g. where the consuming project is named `targets.csproj`:

```
targets: Working directory: foo
targets: dotnet build Boo.sln
```

or `build.csproj`:

```
build: Working directory: foo
build: dotnet build Boo.sln
```

With an optional param to control the program name.

**Comment by adamralph** [comment]
Or should the default prefix by the entry assembly name? 🤔

**Comment by adamralph** [comment]
Released in [6.1.0-beta.1](https://www.nuget.org/packages/SimpleExec/6.1.0-beta.1).
